examples/experimental/dagster-airlift/README.md

```diff
@@ -36,45 +36,49 @@ Airlift depends on the the availability of Airflow’s REST API. Airflow’s RES
 
 # Guide
 
-In the below guide, we'll be working with a sample project, found in [`examples/tutorial-example`](./examples/tutorial-example/).
+The below guide will walk through the process of peering, observing, and migrating assets from an Airflow DAG to Dagster. We'll be working with a sample project, found in [`examples/tutorial-example`](./examples/tutorial-example/), which you can clone and use to follow along.
 
-## Running airflow locally
+## Running Airflow locally
 
-The tutorial example involves running a local airflow instance. This can be done by running the following commands from the root of the `examples/tutorial-example` directory.
+The tutorial example involves running a local Airflow instance. This can be done by running the following commands from the root of the `tutorial-example` directory.
+
+First, install the required python packages:
 
 ```bash
 make airflow_install
 ```
 
-This command installs python packages required for running airflow.
+Next, scaffold the Airflow instance, and initialize the dbt project:
 
 ```bash
 make airflow_setup
 ```
 
-This command scaffolds a local installation of airflow (setting AIRFLOW_HOME env var), and sets up a dbt project.
+Finally, run the Airflow instance with environment variables set:
 
 ```bash
 make airflow_run
 ```
 
-Which runs `airflow standalone` with required env vars set.
+You should now be able to access the Airflow UI at [http://localhost:8080](http://localhost:8080), with the default username and password set to `admin`.
+
+You should be able to see the `rebuild_customers_list` DAG in the Airflow UI, made up of three tasks: `load_raw_customers`, `run_dbt_model`, and `export_customers`.
 
-## Peering
+## Peering Dagster to your Airflow instance
 
-The first step is to peer the Dagster code location and the Airflow instance, which will create an asset representation of each of your Airflow DAGs in Dagster. This process does not require any changes to your Airflow instance.
+The first step is to peer your Airflow instance with a Dagster code location, which will create an asset representation of each of your Airflow DAGs that you can view in Dagster. This process does not require any changes to your Airflow instance.
 
-First, you will need to install `dagster-airlift` in your Dagster environment:
+First, you will need to set up the `dagster-airlift` package in your Dagster environment:
 
 ```bash
 pip install uv
-uv pip install dagster-airlift[core] dagster-webserver
+uv pip install 'dagster-airlift[core]' dagster-webserver
 ```
 
-Next, you should create a `Definitions` object using `build_defs_from_airflow_instance`.
+Next, create a new Python file to hold your Dagster code. Create a `Definitions` object using `build_defs_from_airflow_instance`:
 
+`airlift.py`
```

Comment:
The rendering of this standalone is a little goofy:

![Screenshot 2024-08-27 at 12.09.03 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/wS38mdcdU6aAeJobBdry/91a1df32-ee44-495a-be88-565120d63cf7.png)